web/modules/custom/bnf/bnf_server/src/Plugin/GraphQL/DataProducer/NewContentProducer.php

```diff
@@ -72,16 +71,18 @@ public function resolve(string $termUuid, string $since): NewContentResponse {
     }
 
     $query = $this->nodeStorage->getQuery();
-    $query->condition('created', $since->getTimestamp(), '>')
-      ->condition('status', Node::PUBLISHED);
+    $query->condition('changed', $since->getTimestamp(), '>');
```

Comment:
We're sure that `scheduler` updates `changed` when publishing?

Reply:
I just double checked - yep, it updates `changed` :)